cleanlab/multiannotator.py

```diff
@@ -92,10 +99,16 @@ def get_label_quality_multiannotator(
 
         * ``crowdlab``: an emsemble method that weighs both the annotators' labels as well as the model's prediction.
         * ``agreement``: the fraction of annotators that agree with the consensus label.
+    temp_scale : bool, default = False
```

Comment:
at user-facing level, can we call this arg: `calibrate_probs` or `calibrate_pred_probs` ?  That provides more flexibility in the future to change the algorithm used to perform the calibration